HelloPi/README.md

```diff
@@ -0,0 +1,62 @@
+
+# HelloPi
+
+A simple HelloFX application with Java 11+, JavaFX 11+ and GraalVM that runs on HotSpot or as a native image on Desktop and Embedded devices (Raspberry Pi).
+
+## Documentation
+
+Read about getting started with JavaFX on Embedded [here](https://docs.gluonhq.com/#platforms_embedded)
+
+Note that Gluon applications can run on JVM/HotSpot on ARM devices (32/64 bits) but native images can be created only for AArch64 (64 bits).
+
+## Quick Instructions
+
+We use [GluonFX plugin](https://docs.gluonhq.com/) to build a native image for platforms including desktop, android and iOS.
+Please follow the prerequisites as stated [here](https://docs.gluonhq.com/#_requirements).
+
+### Desktop
+
+Run the application on JVM/HotSpot:
+
+    mvn gluonfx:run
+
+Run the application and explore all scenarios to generate config files for the native image with:
+
+    mvn gluonfx:runagent
+
+Build a native image using:
+
+    mvn gluonfx:build
+
+Run the native image app:
+
+    mvn gluonfx:nativerun
```

Comment:
These steps are for building AND running on the host machine. I think it makes sense to run `runagent` but the other steps are irrelevant here?

Reply:
Well, that's true, but we just describe the necessary steps to test the sample on desktop, which is still convenient before deploying to other platforms (mobile, embedded). Testing and fixing errors is faster on desktop.

Reply:
Totally true, I think it would just be avoiding confusing if you add that (testing and fixing and development in general is best done on desktop) in the README?

Reply:
Done